pkg/services/alerting/service.go

```diff
@@ -77,6 +78,7 @@ func (s *AlertNotificationService) UpdateAlertNotification(ctx context.Context,
 
 	model := models.AlertNotification{
 		Id:       cmd.Id,
+		OrgId:    cmd.OrgId,
```

Comment:
I tried to create a few tests for this but entered the land of diminishing returns so decided to skip. Would rather get the fix in and follow up (if you feel strongly about it) in another PR.